compose.yaml

```
@@ -1,7 +1,9 @@
 ---
-version: "3.7"
+name: admiral
 
 secrets:
+  # TODO: https://github.com/cisagov/admiral/issues/80
+  # move out of secrets
```

Can this comment be clarified a bit?  Maybe something like:
```suggestion
  # TODO: Move config files from secrets to a new configs section
  # See https://github.com/cisagov/admiral/issues/80 for details
```